### ISSUE TITLE
TypeScript "circular definition" error on import

### ISSUE THREAD

**Issue by cshouts-tasc** [vanillaes/csv-es#13]
When importing csv-es into a TypeScript project via `import CSV from 'csv-es` the following errors appear at compile time:

`ERROR in node_modules/csv-es/index.d.ts(28,14): error TS2303: Circular definition of import alias 'parse'.
node_modules/csv-es/index.d.ts(29,14): error TS2303: Circular definition of import alias 'stringify'.`

**Comment by cshouts-tasc** [comment]
It looks like this issue was introduced in v1.4.0.

**Comment by evanplaice** [comment]
The implementation changed in v1.4.0 to allow better tree-shaking support.

Instead of a CSV class with parse and stringify methods. The 2 methods were split into separate functions. This allows bundlers to tree-shake out the unused parts.

To avoid a major breaking change a `default` export was provided to mimic the previous API. The change worked flawlessly in vanilla ES but apparently TS isn't spec compliant in this very specific case.

I removed the `default` export and published the change as v2.0. To use it you'll need to use either

```typescript
import * as CSV from 'csv-es';
```

or the named exports

```typescript
import { parse, stringify } from 'csv-es'
```

Thanks for taking the time to report this issue. I'm going to close this issue, feel free to re-open if you're still having trouble.

**References**

- [Exporting One Value on Both Default and Named Exports (Circular) - StackOverflow](https://stackoverflow.com/q/60212268/290340)
- [Exporting One Value on Both Default and Named Exports (Circular) - microsoft/
TypeScript](https://github.com/microsoft/TypeScript/issues/36774)

**Comment by evanplaice** [comment]
Fixed in [v2.0](https://github.com/vanillaes/csv-es/releases/tag/v2.0.0)
